### PR TITLE
Add feature flag for both platform and sites

### PR DESCRIPTION
**Pull request by OmarIthawi**
Allow the plugin to be used by Tahoe, by letting individual sites disabling the feature.

### CODE REVIEW

**Review by thraxil — APPROVED** [review]

**Comment by OmarIthawi** [comment]
Thanks @thraxil! Will do a bit more testing and ship to Tahoe Staging.

File: README.rst [hunk 1]
```diff
@@ -58,6 +59,8 @@ Next, install LingoX and configure it:
   language for a specific site: go to Sites
   Configuration ``/admin/site_configuration/siteconfiguration/``, add a ``LANGUAGE_CODE`` key with the desired
   site-specific value to the site's configuration JSON.
+  To enable or disable the feature on a specific site, define the variable ``ENABLE_LINGOX`` in the site's
```
Comment (melvinsoft) [inline README.rst:62]: @OmarIthawi We should clarify that the variable goes under the `FEATURES` variable dict.

**Review by melvinsoft — APPROVED** [review]
@OmarIthawi Just a minor change in the doc, feel free to merge after the change.

File: README.rst (hunk 1 above)
Comment (OmarIthawi) [reply README.rst:62]: Not really @melvinsoft, this is for the `Site` configuration in the admin panel. `FEATURES` only exist in the `aws.py` configurations.

What do you think?

Comment (melvinsoft) [reply README.rst:62]: Sorry, you're absolutely right @OmarIthawi. Ship it!

**Comment by OmarIthawi** [comment]
Status: There plugin still don't work reliably on Tahoe yet. However, I'm merging this PR because it is generally in improvement with passing tests.

The [Trello Card](https://trello.com/c/XC2qHgJ3/2550-4-test-microsite-settings-in-lingox) however will stay on Red Team board.

cc: @melvinsoft

**Pull request merged**